packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -1714,12 +1717,15 @@ namespace Tpetra {
     const char tfecfFuncName[] = "insertGlobalIndicesImpl: ";
     const LO lclRow = static_cast<LO> (rowInfo.localRow);
 
+    Details::ProfilingRegion regionIGII("Tpetra::CrsGraph::insertGlobalIndicesImpl");
+
     auto numEntries = rowInfo.numEntries;
     using inp_view_type = View<const GO*, Kokkos::HostSpace, MemoryUnmanaged>;
     inp_view_type inputInds(inputGblColInds, numInputInds);
     size_t numInserted;
     {
       auto gblIndsHostView = this->gblInds_wdv.getHostView(Access::ReadWrite);
+      // FIXME - device
```

Comment:
Does this mean that the code needs to be rewritten to use device?

Reply:
most of the updates that Steve Kennon made were to the matrix.  similar updates can also be made to the graph.